components/director/internal/authenticator/middleware_test.go

```diff
@@ -98,7 +98,27 @@ func TestAuthenticator_Handler(t *testing.T) {
 		//then
 		assert.Equal(t, "OK", rr.Body.String())
 		assert.Equal(t, http.StatusOK, rr.Code)
+	})
+
+	t.Run("Error - forbidden when tenant is empty", func(t *testing.T) {
+		//given
+		middleware := createMiddleware(t, true)
+		handler := testHandler(t, tnt, scopes)
+		rr := httptest.NewRecorder()
+
+		req, err := http.NewRequest("GET", "/", nil)
```

Comment:
This code repeats in many test cases, maybe it is worth to extract it to some helper func?

For example:
```go
func testRequest(t *testing.T) *http.Request {
    req, err := http.NewRequest("GET", "/", nil)
    require.NoError(t, err)

    return req
}
```